catalog-rest-service/src/main/java/org/openmetadata/catalog/security/JwtFilter.java

```diff
@@ -104,10 +103,33 @@ public void filter(ContainerRequestContext requestContext) {
     String tokenFromHeader = extractToken(headers);
     LOG.debug("Token from header:{}", tokenFromHeader);
 
+    DecodedJWT jwt = validateAndReturnDecodedJwtToken(tokenFromHeader);
+
+    Map<String, Claim> claims = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+    claims.putAll(jwt.getClaims());
+
+    String userName = validateAndReturnUsername(claims);
```

Comment:
This can throw an error right? if the token is sent for a bot this will not find the claim in the token and can throw an error?

Reply:
but this endpoint should not be used for bot ..right ?!